app/api/api_v1/endpoints/pm_rent.py

```
@@ -0,0 +1,141 @@
+        reference=payload.reference,
+        notes=payload.notes,
+        receipt_document_id=payload.receipt_document_id,
+    )
+    return RentPaymentSchema.model_validate(payment)
+
+
+@router.post("/charges/{charge_id}/tenant-payment-intent", response_model=RentPaymentSchema)
+async def tenant_payment_intent(
+    charge_id: int,
+    payload: RentPaymentCreate,
+    current_user: UserSchema = Depends(get_current_active_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)